### PR TITLE
[PRO-1575] add endpoint for counting devices in group

### PR DESCRIPTION
**Pull request by rdanitz**

### CODE REVIEW

File: device-registry/src/main/scala/org/genivi/sota/device_registry/GroupMember.scala [hunk 1]
```diff
@@ -40,4 +40,9 @@ object GroupMember {
       .filter(r => r.groupId === groupId)
       .map(r => r.deviceUuid)
       .result
+
+  def countDevicesInGroup(groupId: Uuid)(implicit ec: ExecutionContext): DBIO[Int] =
+    GroupInfoRepository.getGroupInfoById(groupId)
```
Comment (CalumMcCall) [inline device-registry/src/main/scala/org/genivi/sota/device_registry/GroupMember.scala]: Can you indent before the .getGroupInfoById()? Then it matches the rest of the file

File: device-registry/src/test/scala/org/genivi/sota/device_registry/GroupsResourceSpec.scala [hunk 2]
```diff
@@ -155,4 +162,10 @@ class GroupsResourceSpec extends FunSuite
       groups.count(e => e.id.equals(group.id) && e.groupName.equals(newGroupName)) shouldBe 1
     }
   }
+
+  test("counting devices should fail for non-existent groups") {
+    countDevicesInGroup(genUuid.sample.get) ~> route ~> check {
+      status shouldBe NotFound
+    }
+  }
```
Comment (CalumMcCall) [inline device-registry/src/test/scala/org/genivi/sota/device_registry/GroupsResourceSpec.scala]: What about a test which checks the count is correct?

Comment (rdanitz) [inline device-registry/src/test/scala/org/genivi/sota/device_registry/GroupsResourceSpec.scala]: it's part of creating the group; GroupsResourceSpec.scala#100

Comment (CalumMcCall) [inline device-registry/src/test/scala/org/genivi/sota/device_registry/GroupsResourceSpec.scala]: Ah my bad.

**Pull request merged**